Directory.Build.props

```diff
@@ -2,14 +2,14 @@
 
   <PropertyGroup>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
-    <LangVersion>7.3</LangVersion>
+    <LangVersion>8.0</LangVersion>
     <TreatWarningsAsErrors Condition="'$(Configuration)' == 'Release'">true</TreatWarningsAsErrors>
-    <NoWarn>$(NoWarn);CS1591;CA1812;SA1309;SA1600;SA1633;SA1200;SA1202;SA1201;CA1016</NoWarn>
+    <NoWarn>$(NoWarn);CS1591;CA1812;SA1309;SA1600;SA1633;SA1200;SA1202;SA1201;CA1016;CA1303</NoWarn>
   </PropertyGroup>
 
   <PropertyGroup>
     <Description>A library for KMD Logic Identity Authorization</Description>
-    <VersionPrefix>1.0.0</VersionPrefix>
+    <VersionPrefix>1.1.0</VersionPrefix>
```

Comment:
Should we bump this to 2.0.0 since it's a breaking change?

Reply:
Yes